makelib/xpkg.mk

```diff
@@ -19,13 +19,6 @@ ifeq ($(origin XPKG_DIR),undefined)
 XPKG_DIR := $(ROOT_DIR)/package
 endif
 
-# xref https://github.com/upbound/provider-aws/pull/647, https://github.com/upbound/up/pull/309
-# in up v0.16.0, support for ProviderConfig documentation via object annotations was added.
-# by convention, we will assume the extensions file resides in the package directory.
-ifeq ($(origin XPKG_AUTH_EXT),undefined)
-XPKG_AUTH_EXT := $(XPKG_DIR)/auth.yaml
-endif
```

Comment:
I do not quite understand what this is for. but does not seem to be supported by the crossplane-cli

Reply:
I don't recall the history here. 🤔 

@turkehnf @sergenyalcin do you know if removing this will be a problem for upjet providers?